docs/backend/data-update-scripts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Data Update Scripts
+---
+
+## What are Data Update Scripts?
+
+Data Update Scripts were introduced recently in
+[this PR](https://github.com/thepracticaldev/dev.to/pull/6025) and allow us to
+run any data updates we might need. For example, if we added a column to the
+database and then wanted to backfill that column with data, rather than going
+and manually doing it in a console, we would use a DataUpdateScript. Another
+example might be adding a new attribute to Elasticsearch. We could then use a
+DataUpdateScript to reindex all of our models.
+
+## How it works
+
+First off, we added a
+[DataUpdateScript model](https://github.com/thepracticaldev/dev.to/blob/master/app/models/data_update_script.rb)
+to Rails and a corresponding database table. This table is what we use to keep
+track of what scripts have been run and which ones have not/still need to be.
+
+To create a script you can use our custom Rails generator:
+
+```
+rails generate data_update BackfillColumnForArticles
```

Comment:
cool!!

Reply:
+1 to @citizen428 for this one! First time ever making a generator myself!